scripts/run-proxies.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -euo pipefail
+
+: "${REMOVE_CONTAINERS_ON_START:=}"
+: "${REMOVE_CONTAINERS_ON_FAIL:=}"
+: "${REMOVE_CONTAINERS_FORCE:=}"
+: "${RESTART_INTERVAL_TIME:=3}"
+
+containerer() {
+  local builder_cmd
+  if command -v podman >/dev/null 2>&1; then
+    builder_cmd="podman"
+  elif command -v docker >/dev/null 2>&1; then
+    builder_cmd="docker"
+  else
+    echo "Please install podman or docker"
+    exit 1
+  fi
+  $builder_cmd "$@"
+}
+
+ARGS=()
+
+parse_cli() {
+  while [ $# -gt 0 ]; do
+    local arg="$1"
+    local opt="${2:-}"
+    case $arg in
+    -f | --remove-containers | --force)
+      REMOVE_CONTAINERS_ON_START=true
+      REMOVE_CONTAINERS_ON_FAIL=true
+      REMOVE_CONTAINERS_FORCE=true
+      ;;
+    -s | --remove-containers-on-start)
+      REMOVE_CONTAINERS_ON_START=true
+      ;;
+    -x | --remove-containers-on-fail)
+      REMOVE_CONTAINERS_ON_FAIL=true
+      ;;
+    -i | --interval)
+      RESTART_INTERVAL_TIME="${opt}"
+      shift
+      ;;
+    --)
+      shift
+      ARGS+=("$@")
+      break
+      ;;
+    *)
+      ARGS+=("$1")
+      ;;
+    esac
+    shift
+  done
+}
+
+parse_cli "$@"
+
+main() {
+  for proxy in es soffia xroad redis db; do
```

Comment:
May nice to allow user to specify which proxies if it is known what proxies are required to save resources. Default to all if none is specified.

Reply:
Updated :heavy_check_mark: